engine/src/gfx/radar/radar.cpp

```diff
@@ -12,21 +12,21 @@
 namespace Radar
 {
 
-std::auto_ptr<Display> Factory(Type::Value type)
+std::unique_ptr<Display> Factory(Type::Value type)
```

Comment:
These might need to be `std::shared_ptr`; not sure. Does it run well with `std::unique_ptr`?